src/gui/properties/propertieswidget.cpp

```diff
@@ -93,11 +93,13 @@ PropertiesWidget::PropertiesWidget(QWidget *parent)
     m_contentFilterLine->setPlaceholderText(tr("Filter files..."));
     m_contentFilterLine->setFixedWidth(Utils::Gui::scaledSize(this, 300));
     connect(m_contentFilterLine, &LineEdit::textChanged, this, &PropertiesWidget::filterText);
-    m_ui->contentFilterLayout->insertWidget(3, m_contentFilterLine);
+    m_ui->contentFilterLayout->insertWidget(5, m_contentFilterLine);
 
     // SIGNAL/SLOTS
     connect(m_ui->selectAllButton, &QPushButton::clicked, m_propListModel, &TorrentContentFilterModel::selectAll);
     connect(m_ui->selectNoneButton, &QPushButton::clicked, m_propListModel, &TorrentContentFilterModel::selectNone);
+    connect(m_ui->expandAllButton, &QPushButton::clicked, m_ui->filesList, &TorrentContentTreeView::expandFileList);
```

Comment:
Why don't you connect to `expandAll` directly?
And `collapseAll`?

Reply:
Fixed.